server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/core/Capability.scala

```diff
@@ -161,16 +164,41 @@ final case class EhloArg(value: String) extends AnyVal
 final case class EhloArgs(values: List[EhloArg]) extends AnyVal
 
 final case class SubmissionCapability(identifier: CapabilityIdentifier = EMAIL_SUBMISSION,
-                                      properties: SubmissionProperties = SubmissionProperties()) extends Capability
+                                      properties: SubmissionProperties) extends Capability
 
-case object SubmissionCapabilityFactory extends CapabilityFactory {
+case object SubmissionCapabilityFactory {
+  val maximumDelays = Duration.ofDays(1)
+}
+
+final case class SubmissionCapabilityFactory(clock: Clock, supportsDelaySends: Boolean) extends CapabilityFactory {
   override def id(): CapabilityIdentifier = EMAIL_SUBMISSION
 
-  override def create(urlPrefixes: UrlPrefixes): Capability = SubmissionCapability()
+  override def create(urlPrefixes: UrlPrefixes): Capability =
+    if (supportsDelaySends) {
+      advertiseDelaySendSupport
+    } else {
+      advertiseNoDelaySendSupport
+    }
+
+  private def advertiseDelaySendSupport = {
+    val dateAsString = DateTimeFormatter.ISO_INSTANT.withZone(ZoneId.of("UTC")).format(clock.instant().plus(maximumDelays))
+
+    SubmissionCapability(EMAIL_SUBMISSION,
+      SubmissionProperties(MaxDelayedSend(maximumDelays.toSeconds.toInt),
+        Map(EhloName("FUTURERELEASE") -> EhloArgs(List(EhloArg(maximumDelays.toSeconds.toString()), EhloArg(dateAsString))))))
+  }
+
+  private def advertiseNoDelaySendSupport =
+    SubmissionCapability(EMAIL_SUBMISSION,
+      SubmissionProperties(MaxDelayedSend(0),
+        Map()))
+
+  def create(maxDelayedSend: MaxDelayedSend, submissionExtensions: Map[EhloName, EhloArgs]): Capability =
+    SubmissionCapability(EMAIL_SUBMISSION, SubmissionProperties(maxDelayedSend, submissionExtensions))
 }
 
-final case class SubmissionProperties(maxDelayedSend: MaxDelayedSend = MaxDelayedSend(0),
-                                      submissionExtensions: Map[EhloName, EhloArgs] = Map()) extends CapabilityProperties {
+final case class SubmissionProperties(maxDelayedSend: MaxDelayedSend = MaxDelayedSend(86400),
```

Comment:
Can we leverage the `SubmissionCapabilityFactory.maximumDelays` constant instead of an 86400 number?